locales/en/app.ftl

```diff
@@ -675,3 +675,25 @@ promo-fx-mobile-headline = Privacy and speed on mobile
 promo-fx-mobile-body = 
   The { -brand-name } browser is super fast, private by default, and blocks 2,000+ online trackers.
 promo-fx-mobile-cta = Get { -brand-name } Mobile
+
+promo-lockwise-headline = Take your passwords everywhere
+promo-lockwise-body =
+  Use { -brand-lockwise } to keep track of all your passwords. 
+  Access them securely from your computer, phone, or tablet.
+promo-lockwise-cta = Get { -brand-lockwise }
+
+promo-fpn-headline = Keep your real location under wraps
+promo-fpn-body = { -brand-name } 
+  Private Network hides your location to throw off the websites 
```

Comment:
I think "Firefox Private Network" needs to be defined as a brand of its own this time. As far as I know, that stays in English as one name.